.github/workflows/nightly_run.yml

```
@@ -0,0 +1,47 @@
+name: gccrs-testing-nightly-run
+
+on:
+  schedule:
+    - cron: '0 2 * * *' # Run at 2 AM UTC. Does not really matter
+
+jobs:
+  run-testsuites:
+    runs-on: self-hosted
+    container: philberty/gccrs:latest
+    strategy:
+      matrix:
+        testsuite: [rustc-dejagnu, gccrs-parsing, gccrs-rustc-success, gccrs-rustc-success-no-std, gccrs-rustc-success-no-core, blake3, libcore-1.49]
+    steps:
+      - name: Fetch dependencies
+        run: |
+          # FIXME: Switch to `cargo install ftf` on next release, we need the
+          # `result-fmt` feature
+          cargo install --git https://github.com/cohenarthur/ftf
+
+      - uses: actions/checkout@v2
+      - name: Generate testsuites (nightly)
```

```suggestion
      - uses: actions/checkout@v3

      - name: Generate testsuites (nightly)
```

There's v3 of the checkout action IIRC. And lets insert newlines between steps.